cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -241,8 +248,16 @@ function move_sriov_pfs_netns_to_node {
 
     ip link set "$pf_name" netns "$node"
     pf_array+=("$pf_name")
+    if [ "${#pf_array[@]}" -eq "$num_pfs_required" ]; then
```

Comment:
Please drop the `if`.

Reply:
In all places that you asked please drop the if you mean to do as you suggested here i guess
https://github.com/kubevirt/kubevirtci/pull/510#discussion_r550162182